Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.h

```diff
@@ -29,9 +29,7 @@ typedef struct _MSFT_nxOMSAutomationWorkerResource /* extends OMI_BaseResource *
     MI_Instance __instance;
     /* OMI_BaseResource properties */
     /* MSFT_nxOMSAutomationWorkerResource properties */
-    /*KEY*/ MI_ConstStringField WorkspaceId;
```

Comment:
We haven't gotten rid of workspace ID, it has been restructured into a JSON string that contains all settings. The underlying partial support for multi-homing still remains the same.